### ISSUE TITLE
input files required for running code

### ISSUE THREAD

**Issue by thisisnottaken** [ray0809/pix2pix#3]
sir ,
QUE 1 ) https://github.com/ray0809/pix2pix/blob/master/data.py needs image files (jpg and png) in folder /home/ray/python_code/keras_examples/pix2pix/CMP_facade_DB_base/base .
How can I get those ?
QUE 2 ) My understanding is that parameters of neural network are learnt when we have pair of files labelled as corresponding input and output . After training we give an unknown input image and neural network generates output image using parameters it has learnt earlier . Is that alright ?
That leads me to think that pic folder should have atleast 2 files (jpg , one input and one output) 
Target folder can have 1 png file if we desire to predict the output for this input png image .
Is that ok ?

Will wait for your reply.
Thanks for reading.

**Comment by ray0809** [comment]
Hi,
    the jpg and png correspon original picture and label,you can download the dataset to help you to understand the code in data.py[CMP Facade Dataset](http://cmp.felk.cvut.cz/~tylecr1/facade/).
    As you said ,the learned parameters are used for predict the label of input,and here input is jpg

**Comment by thisisnottaken** [comment]
ok .
I will download http://cmp.felk.cvut.cz/~tylecr1/facade/CMP_facade_DB_base.zip and try to understand .

**Comment by thisisnottaken** [comment]
sir , 
Thanks for replying.
Now I understand that , I should put real images in pic npy file and corresponding sketches in target npy file to train the neural network.
After training , our goal is to make a real image from sketch (in this case).
I think , you save unknown sketch as first image in target.npy . I assume the first image in pic.npy must be a blank image .The corresponding improved successive real image is generated after first , 50th and 100th epoch .
Is that correct ?

thanks.

**Comment by ray0809** [comment]
Hi,
    the trained model here is used for changing sketch to real one.the pic.npy is real one,target.npy is sketch.
    there is no blank image in npy,the images in dataset are pair

**Comment by Srijita173** [comment]
Hi @ray0809  , @thisisnottaken  ,
After training the network with pic and target pairs how do we get the output picture for a new sketch ?
For example, I have trained with the CMP_facade_DB_base dataset pairs . Now if I want to test for a new sketch how do I do that ?

**Comment by kerorojason** [comment]
You can run:
result = g.predict(img_array)
However, after trained on my custom dataset,
I got this image after prediction in every input data...
![download](https://user-images.githubusercontent.com/15242893/51089250-2176fe80-17a5-11e9-8a69-3d7716eab819.png)
